### PR TITLE
Make it so the OB beacons always have squad or human name tied to them.

### PR DESCRIPTION
**Pull request by lbnesquik**
## About The Pull Request
Make it so the OB beacons always have squad or human name tied to them. Like  
`
Orbital beacon (Mandalore gaming)
`
## Why It's Good For The Game
Hopefully make it easier to avoid some friendly fire accidents.

## Changelog
:cl:
add: Added a new way to ID the beacons
/:cl:

### CODE REVIEW

File: code/game/objects/machinery/squad_supply/supply_beacon.dm [hunk 1]
```diff
@@ -101,6 +102,10 @@
 		squad = H.assigned_squad
 		name += " ([squad.name])"
 		squad.squad_orbital_beacons += src
+	else	//So we can just get a goshdarn name.
+		alt_name = H.name
+		name += " ([alt_name])"
+		//alt_name.squad_orbital_beacons += src
```
Comment (BraveMole) [inline code/game/objects/machinery/squad_supply/supply_beacon.dm:108]: early return needed : 
if(H.assigned_squad)
		squad = H.assigned_squad
		name += " ([squad.name])"
		squad.squad_orbital_beacons += src
		return
alt_name = H.name
name += " ([alt_name])"

File: code/game/objects/machinery/squad_supply/supply_beacon.dm [hunk 2]
```diff
@@ -101,6 +102,10 @@
 		squad = H.assigned_squad
 		name += " ([squad.name])"
 		squad.squad_orbital_beacons += src
+	else	//So we can just get a goshdarn name.
+		alt_name = H.name
+		name += " ([alt_name])"
```
Comment (BraveMole) [inline code/game/objects/machinery/squad_supply/supply_beacon.dm:107]:
```suggestion
		name += " ([H])"
```

File: code/game/objects/machinery/squad_supply/supply_beacon.dm [hunk 3]
```diff
@@ -92,6 +92,7 @@
 	icon_activated = "motion1"
 	///The squad this OB beacon belongs to
 	var/datum/squad/squad = null
+	var/alt_name = null
```
Comment (BraveMole) [inline code/game/objects/machinery/squad_supply/supply_beacon.dm:95]: uneeded

**Review by BraveMole — CHANGES_REQUESTED** [review]

**Comment by BraveMole** [comment]
Also, how bout the name of the area rather than the squad or human name?

**Comment by lbnesquik** [comment]
> 
> 
> Also, how bout the name of the area rather than the squad or human name?

Except if we give exact coordinates (which i feel might be a bit too much), areas wont help that much IMO.

**Review by SplinterGP — APPROVED** [review]

**Pull request merged**